packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -66,6 +98,22 @@ abstract class ProgressIndicator extends StatefulWidget {
     super.debugFillProperties(properties);
     properties.add(PercentProperty('value', value, showName: false, ifNull: '<indeterminate>'));
   }
+
+  Widget _buildSemanticsWrapper({
+    @required BuildContext context,
+    @required Widget child,
+  }) {
+    String expandedSemanticsValue = semanticsValue;
+    if (value != null) {
+      expandedSemanticsValue ??= '${(value * 100).round()}%';
+    }
+    return Semantics(
+      container: true,
```

Comment:
Does this have to be a container then? If it's not a container a dev could easily turn this into a live region by wrapping it in a semantics widget.

Reply:
How would that handle if this indicator was the child of something that also set label and/or value?

Reply:
If the other child has a semantics configuration that's incompatible with this one, separate SemanticsNodes will get created.

Reply:
Sounds good, I'll remove it.